src/commands/upload-graphql-shema.js

```diff
@@ -0,0 +1,71 @@
+const FaunaCommand = require('../lib/fauna-command.js')
+const { flags, Command } = require('@oclif/command')
+const fetch = require('node-fetch')
+const fs = require('fs')
+const path = require('path')
+const { errorOut } = require('../lib/misc.js')
+
+class UploadGraphQLSchemaCommand extends FaunaCommand {
+  allowedExt = ['.graphql', '.gql']
+
+  async run() {
+    try {
+      const { graphqlPath } = this.args
+      const { mode, graphqlHost } = this.flags
+
+      if (!this.allowedExt.includes(path.extname(graphqlPath))) {
+        errorOut(
+          'You must provide a Graphql file (`.graphql` or `.gql`) to upload a schema'
+        )
+      }
+
+      const {
+        connectionOptions: { secret },
+      } = await this.getClient()
+
+      console.info(`UPLOADING SCHEMA (mode=${mode}): ${graphqlPath}`)
+      const text = await fetch(`${graphqlHost}/import?mode=${mode}`, {
```

Comment:
Can we trim trailing whitespace and slashes from the URL provided for the `graphqlHost`?